source/includes/_quickstart-wallets.md

```diff
@@ -0,0 +1,42 @@
+## <em>Quickstart Wallets</em> 🚧🚧
+
+This guide gives a brief overview to how to do the following, using the wallet API:
+
+- Register parties
+- Create wallets
+- Registering a location outside the system to send money to
+- Make a outbound payment
+
+Please ensure you also read the complete documentation for each API.
+
+All money held in wallets is eMoney and therefore might not be suitable for all use cases. Let us know if you have questions.
+
+- [Payments API](/#payments-managers)
+
+## Registering companies
+
+In our payments system we refer to all different types of entity as a Party. In the case of [registering a company](/#payments-manager-post-companies) you will receive a `partyId` in the response. This will look something like `COM~d28360c5-07a3-4d78-ade4-bddcdd8b5502` and is our reference to that entity which will be used to create wallets under that party.
+
+It's worth noting that money in wallets underneath this party will be held in their name. For example a if we registered Goji under the wallet API, money held in any wallet under that party would be owned by Goji. Likewise if we registered a Borrower in the P2P space, any money in wallets under that borrower is owned by the Borrower.
+
+Registering new parties is an asyncronous process therefore the party will not be ready for use straight away. You will receive a [webhook](/#webhooks-company_registration_update) when the party is enabled for use.
+
+## Creating wallets
+
+Once you've received the webhook from the previous stage letting you know the party is registered, we will be ready to create wallets for that Party.
```

Comment:
`we will be ready to create wallets for that Party`
Shouldn't this be `you will...`?
Or `wallets can be created for that Party`?

Also should the `Party` be capitalise here? We use lower case in other places.

Reply:
Ha clearly drifted into siding with the client a bit too much. Too much rushing trying to get it out for Nester...